### PR TITLE
WFLY-19021, community doc. Document usage of Galleon stability properties

### PR DESCRIPTION
**Pull request by jfdenise**
That is  a piece of the community doc for: https://issues.redhat.com/browse/WFLY-19021

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

____

### CODE REVIEW

**Comment by bstansberry** [comment]
@jfdenise I think we need to cover the config-stability-level and package-stability-level and mention what happens if none of these are set. The text here implies that you can't have lower stability features if you don't set this, but they are available.

File: docs/src/main/asciidoc/Bootable_Guide.adoc [hunk 1]
```diff
@@ -135,6 +135,43 @@ If your project is using link:WildFly_and_WildFly_Preview{outfilesuffix}[WildFly
 is ``wildfly-preview@maven(org.jboss.universe:community-universe)``.
 ====
 
+[[wildfly-jar-maven-plugin-galleon-configuration]]
+=== Setting a stability level when building a Bootable JAR
+
+In order to support xref:Admin_Guide.adoc#Feature_stability_levels[WildFly Stability levels], Galleon defines
```
Comment (bstansberry) [inline docs/src/main/asciidoc/Bootable_Guide.adoc:141]: s/WildFly Stability levels/WildFly feature stability levels/g

**Review by bstansberry — CHANGES_REQUESTED** [review]
@jfdenise Besides my general comment above, there's another minor point.

File: docs/src/main/asciidoc/Bootable_Guide.adoc (hunk 1 above)
Comment (jfdenise) [reply docs/src/main/asciidoc/Bootable_Guide.adoc]: Thanks fixed.

**Comment by jfdenise** [comment]
@bstansberry , I think I covered the missing aspects.

File: docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc [hunk 2]
```diff
@@ -114,6 +114,47 @@ A default configuration is identified by _<configuration model>_/_<configuration
...
+
+If no stability level is provided, the WildFly feature-pack default stability levels for both configurations (`community`) 
+and packages (`experimental`) apply. When provisioning WildFly with no stability level set, 
+the `community` and `default` features are provisioned, the `preview` and `experimental` features are ignored. 
+The `experimental`, `preview`, `community` and `default` packages are provisioned.
+
+[NOTE]
+====
+Having JBoss Modules modules at a lower stability level present in the installation 
+allows you to modify a provisioned server configuration to contain extensions and subsystems at a lower stability level
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc:136]: Modules can be associated with individual resources below the subsystem level, so I think

"extensions and subsystems or finer grained configuration elements that are at a lower stability level..."

**Review by bstansberry — CHANGES_REQUESTED** [review]
@jfdenise Other than one small point, LGTM. Thanks!

File: docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc (hunk 2 above)
Comment (jfdenise) [reply docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc]: @bstansberry , thank-you, I added your suggestion.

**Comment by bstansberry** [comment]
Thanks @jfdenise

**Pull request merged**